apps/complaints/index.js

```diff
@@ -157,6 +157,8 @@ module.exports = {
       }
     },
     '/premium-service-centre': {
+      next: '/complaint-details',
+      fields: ['service-centre-city'],
```

Comment:
they don't _need_ to be, it would be nice though, all these 2 bytes add up - you know the saying... take care of the pennies...